virt/logging_events_monitoring/virt-prometheus-queries.adoc

```diff
@@ -30,6 +25,8 @@ include::modules/monitoring-querying-metrics-for-user-defined-projects-as-a-deve
 
 include::modules/virt-querying-metrics.adoc[leveloffset=+1]
 
+include::modules/virt-live-migration-metrics.adoc[leveloffset=+1]
+
 [id="additional-resources_virt-prometheus-queries"]
 [role="_additional-resources"]
```

Comment:
I think it would be helpful to add these links to Additional resources:

https://prometheus.io/docs/prometheus/latest/querying/basics/

https://prometheus.io/docs/prometheus/latest/querying/examples/